wiki/Interface/en.md

```diff
@@ -1,227 +1,292 @@
-<img src="Intro_static.jpg" title="fig:Right|Introduction screen" alt="Right|Introduction screen" width="400" /> \_\_TOC\_\_
-=Main Menu= <img src="introscreen.png" title="fig:Main menu" alt="Main menu" width="280" />
+<!--Logo definitions-->
+[Osu!]: /wiki/shared/Osu.gif
+[Taiko]: /wiki/shared/Taiko.gif
+[Ctb]: /wiki/shared/Ctb.gif
+[Mania]: /wiki/shared/Mania.gif
+[Heart]: /wiki/shared/Heart.gif
+[Fire]: /wiki/shared/Fire.gif
 
-The \[1\] osu! logo pulses according to the [BPM](Timing "wikilink") of any song currently playing on the Intro screen. If no song is playing, it pulses at a default slow rate. The keyboard shortcut is in parentheses.
+# Interface
 
--   \[2\] Click Play (P) or the logo to switch to the second intro screen.
--   \[3\] Click Edit (E) to open the song selection Screen for the [Editor](Editor "wikilink").
--   \[4\] Click Options (O) to go to the [Options](Options "wikilink") Screen.
--   \[5\] Click Exit (Esc) to exit osu!.
+![Introduction screen](img/Intro_static.jpg)
+
+## Main Menu
+
+![Main Menu](img/Introscreen.png)
+
+The \[1\] [osu!cookie](/wiki/Glossary/#cookie) pulses according to the BPM of any song currently playing on the main menu. In addition, bars will extend out of the osu!cookie in accordance to the song's volume. If no song is playing, it pulses at a slow 60 BPM. The elements of the main menu are as follows:
```

Comment:
Can you switch the name and number? It looks better as `osu!cookie [1]`. Same with everything on this section.